_posts/2018-01-25-review-things-3.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Review: Things 3 for Mac and iOS"
+subtitle: A task manager you will actually want to use
+custom_type: post
+review:
+  rating: 4.5
+  type: app
+  app:
+    developer: Cultured Code
+  price: 9.99-$49.99
+  vendor: Mac App Store
+  url: https://itunes.apple.com/us/app/things-3/id904280696?mt=12
+---
+
+{% figure extendout /uploads/2018/01/things-review-01.jpg "Things 3 on iPad." %}
+
+I've never been a fan of todo apps. I just don't use them. People everywhere seem to swear by these things. There's much talk about the whole GTD mentality, and what app was the best for getting things done.
+
+{% include toc.html %}
+
+I came across Things years ago. I owned the first version of the app, but didn't end up using it all that much. Recently someone I know started using Things 3 and I was intrigued. I'd been using Trello to keep track of my ideas for articles on this site, video ideas, and jobs that I'd applied to. I have a love/hate relationship with Trello. I feel that parts of the app are very well designed, and others not so much.
+
+After watching and reading some positive reviews, I decided to give Things 3 a try. Cultured Code offers a fifteen-day trial of the Mac app.
+
+I was hooked. Things 3 is the first todo app that I actually use. I attribute it in part to the design, but also to where I am in life. These days, I feel I forget things a lot more and need to write down things I need to get done.
+
+## First Impressions
+
+- I'm delighted by how much of a facelift was given to the UI.
+- Areas and Projects are exactly what I want from a to-do app.
+- There is a wonderful satisfaction in looking at the Logbook and seeing all the things you've accomplished.
+
+## Design
+
+Cultured Code has hit it out of the park. The design of the Mac app, the iPad app, and the iPhone app are nothing less than superb. The Apple Watch app is excellent too, which I'll touch on a little bit later.
+
+Gone are the days of the pale colors, and a design reminiscent of skeuomorphism. The new interface sports brightly-colored icons, revamped typography, and beautiful use of white space. While the aesthetic and experience is consistent across platforms, Cultured Code has done a wonderful job of tailoring each interface to get the most out of each operating system.
+
+Cultured Code has achieved an incredible feat hear. These are four different apps it's made you believe are one. And in contrast to some where the functionality is drastically reduced on iOS, I could never touch my Mac again and still have all the same control of my to-dos.
```

Comment:
> And in contrast to some where the functionality

Some what? Perhaps something more like this anyway:

"And in contrast to those apps where functionality is drastically reduced on iOS"